docs/how-to/hyperparameter-tuning.txt

```diff
@@ -48,31 +49,35 @@ and their associated options.
 Instrumenting Model Code
 ------------------------
 
-Determined injects hyperparameters from the experiment configuration into model
-code via a context object in the Trial base class. This :class:`TrialContext
-<determined.TrialContext>` object exposes a ``get_hparam`` method that takes
-the hyperparameter name.  At trial runtime, Determined injects a value for the
-hyperparameter. For example, to inject the previous section's
-``dropout_probability`` into the constructor of a PyTorch `Dropout
+Determined injects hyperparameters from the experiment configuration
+into model code via a context object in the Trial base class. This
+:class:`TrialContext <determined.TrialContext>` object exposes a
+:func:`get_hparam <determined.TrialContext.get_hparam>` method that
+takes the hyperparameter name.  At trial runtime, Determined injects a
+value for the hyperparameter. For example, to inject the value of the
+``dropout_probability`` hyperparameter defined above into the
+constructor of a PyTorch `Dropout
 <https://pytorch.org/docs/stable/nn.html#dropout>`_ layer:
 
 .. code:: python
 
   nn.Dropout(p=self.context.get_hparam("dropout_probability"))
 
-To see hyperparameter injection throughout a complete Trial implementation:
-:ref:`pytorch-mnist-tutorial`.
+To see hyperparameter injection throughout a complete Trial
+implementation, refer to the :ref:`pytorch-mnist-tutorial`.
+
+Specifying the Search Algorithm
+-------------------------------
 
-Specifying the Searcher
------------------------
-The model developer configures a searcher to implement one of many
-:ref:`supported hyperparameter tuning algorithms<topic-guides_hp-tuning-det>`.
-Aside from the ``single`` searcher, a searcher runs multiple trials and decides
-the hyperparameter values to use in each trial.  Every searcher requires the
-optimization objective ``metric`` field in addition to searcher-specific
-options. For instance, the ``adaptive_simple`` searcher implements adaptive
-downsampling given the maximum number of trials to run and the maximum number of
-training steps allowed per trial:
+Determined supports a :ref:`variety of hyperparameter search algorithms
+<topic-guides_hp-tuning-det>`.  Aside from the ``single`` searcher, a
+searcher runs multiple trials and decides the hyperparameter values to
+use in each trial.  Every searcher is configured with the name of the
```

Comment:
nit: It looks like there are double spaces on "...each trial.  Every searcher..." and "<topic-guides_hp-tuning-det>`. Aside" .

Reply:
We use single vs. double spaces in various places throughout the docs. Would be nice to be more consistent, although it shouldn't make any difference in the rendered doc output.